tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php

```diff
@@ -643,7 +643,7 @@ public function testSetDefaultFetchMode()
         $stmt->setFetchMode(\PDO::FETCH_NUM);
 
         $row = array_keys($stmt->fetch());
-        self::assertEquals(0, count( array_filter($row, function($v) { return ! is_numeric($v); })), "should be no non-numerical elements in the result.");
+        self::assertCount(0, array_filter($row, function($v) { return ! is_numeric($v); }), "should be no non-numerical elements in the result.");
```

Comment:
If this test fails, then the failure message doesn't tell what exactly went wrong (i.e. which key is non-numeric). Besides that, `array_filter()` internally performs `if` which is not a good thing for a test. Instead, given the set of selected columns is known and static, we should make an explicit assertion on that like:
```php
self::assertSame([0, 1], $keys);
```

Reply:
Can I ask you to propose this changes in another PR, after this one is merged? So we can keep one line os changes per PR :sweat_smile:

Reply:
You can leave it as is if you want. Not sure if I'm going to propose this change anytime soon.